src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -142,3 +128,50 @@ $card-left-offset: 40px;
         margin-left: 0;
     }
 }
+
+.bcs-task-assignment-tooltip-text {
+    text-align: center;
+}
+
+.bcs-task-assignment-status {
+    display: inline-block;
+    height: 32px;
+    position: relative;
+    width: 32px;
+
+    & + .bcs-task-assignment-status,
+    & + .bcs-task-assignment-count-container {
+        margin-left: $assignment-avatar-spacing;
+    }
+}
+
+.bcs-task-assignment-status-icon {
+    @include avatar-badge-icon;
+}
+
+.bcs-task-assignment-list {
+    max-width: 100%;
+    width: 280px;
```

Comment:
Can this be avoided by making the overlay aware of its parent's size?

Reply:
This is to make them all the same size matching the design, unless it would cause the container to scroll.